webservices/common/models/totals.py

```diff
@@ -35,6 +35,42 @@ class CommitteeTotals(BaseModel):
     last_debts_owed_by_committee = db.Column(db.Numeric(30, 2))
     last_debts_owed_to_committee = db.Column(db.Numeric(30, 2))
 
+class CandidateCommitteeTotals(db.Model):
+    __abstract__ = True
+    #making this it's own model hieararchy until can figure out
+    #how to maybe use existing classes while removing primary
+    #key stuff on cycle
```

Comment:
I'm not sure I follow about removing the primary key stuff on cycle in the context of using existing classes, could you expand on this a bit?  Are you referring to other existing models we already have defined and leveraging those, or because of the `full_election` primary key defined on the models inheriting this base class of sorts?

While I'm a bit confused by this, I'm also not seeing anything wrong here in the structure as defined so this isn't a blocker!